tools/release

```diff
@@ -51,17 +51,19 @@ if 'upload' not in sys.argv:
     sys.exit(0)
 
 # Register with the Python Package Index (PyPI)
-print( 'Registering with PyPI...')
+print( 'Not Registering with PyPI, registering with setup.py is insecure as communication is not encrypted')
```

Comment:
I don't think we need to print a message saying that we're not doing this. A comment is fine. And the call can be removed rather than just commented.